config/routes.rb

```diff
@@ -1,5 +1,5 @@
 MANIACChess::Application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
I just resisted one the resources I used. From what I understand, this line is saying not to use the Devise namespace for the omniauth_callbacks controller. 
https://youtu.be/X6tKAUOMzCs?t=2m30s